docs/CreateDialog.md

```diff
@@ -57,21 +68,178 @@ In the related `<Resource>`, you don't need to declare a `create` component as t
 
 **Note**: You can't use the `<CreateDialog>` and have a standard `<Edit>` specified on your `<Resource>`, because the `<Routes>` declarations would conflict. If you need this, use the [`<CreateInDialogButton>`](./CreateInDialogButton.md) instead.
 
-`<CreateDialog>` accepts the same props as the [`<Create>`](./Create.md) component, and the same type of children (e.g. a [`<SimpleForm>`](./SimpleForm.md) element).
-
-* `children`: the components that renders the form
-* `className`: passed to the root component
-* [`component`](./Create.md#component): override the root component
-* [`disableAuthentication`](./Create.md#disableauthentication): disable the authentication check
-* [`mutationOptions`](./Create.md#mutationoptions): options for the `dataProvider.create()` call
-* [`record`](./Create.md#record): initialize the form with a record
-* [`redirect`](./Create.md#redirect): change the redirect location after successful creation
-* [`resource`](./Create.md#resource): override the name of the resource to create
-* [`sx`](./Create.md#sx-css-api): Override the styles
-* [`title`](./Create.md#title): override the page title
-* [`transform`](./Create.md#transform): transform the form data before calling `dataProvider.create()`
-
-Check [the `ra-form-layout` documentation](https://marmelab.com/ra-enterprise/modules/ra-form-layout#createdialog-editdialog--showdialog) for more details.
+
+## Props
+
+`<CreateDialog>` accepts the following props:
+
+| Prop           | Required | Type              | Default | Description |
+| -------------- | -------- | ----------------- | ------- | ----------- |
+| `children`     | Required | `ReactNode`       |         | The content of the dialog. |
+| `fullWidth`    | Optional | `boolean`         | `false` | If `true`, the dialog stretches to the full width of the screen. |
+| `maxWidth`     | Optional | `string`          | `sm`    | The max width of the dialog. |
+| `mutation Options` | Optional | `object`       |         | The options to pass to the `useMutation` hook. |
+| `resource`     | Optional | `string`          |         | The resource name, e.g. `posts`
+| `sx`           | Optional | `object`          |         | Override the styles applied to the dialog component. |
+| `transform`    | Optional | `function`        |         | Transform the form data before calling `dataProvider.update()`. |
+
+## `children`
+
+`<CreateDialog>` doesn't render any field by default - it delegates this to its children, usually a Form component.
+
+React-admin provides several built-in form layout components:
+
+- [`SimpleForm`](./SimpleForm.md) for a single-column layout
+- [`TabbedForm`](./TabbedForm.md) for a tabbed layout
+- [`AccordionForm`](./AccordionForm.md) for long forms with collapsible sections
+- [`LongForm`](./LongForm.md) for long forms with a navigation sidebar
+- [`WizardForm`](./WizardForm.md) for multi-step forms
+- [`CreateDialog`](./CreateDialog.md) for sub-forms in a modal dialog
```

Comment:
I think is a bit weird to mention the component itself in the `children` section